specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-10-01/examples/ManagedNetwork/createOrUpdateRule.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "test-rg",
+    "workspaceName": "aml-workspace-name",
+    "ruleName": "rule-name",
+    "api-version": "2023-10-01",
+    "body": {
+      "properties": {
+        "type": "FQDN",
+        "destination": "destination_endpoint",
+        "status": "Active",
+        "category": "UserDefined"
+      }
+    }
+  },
+  "responses": {
+    "202": {
```

Comment:
why doesn't create or update have an example 201 response?

Reply:
Our service structure returns 202+ location header for all long-running operations. I don't think that's something we can change easily.